### PR TITLE
Feature/game stats bb

### PR DESCRIPTION
**Pull request by billyburch**
This PR will reflect two methods to be added to the Game class - highest_total_score, & lowest_total_score. Additionally, you will find two tests for these methods in the spec file.

### CODE REVIEW

File: lib/stat_tracker.rb
Comment (EricBelongea) [inline lib/stat_tracker.rb:1]: Productive group merge conflict.

**Review by EricBelongea — APPROVED** [review]

File: lib/stat_tracker.rb
Comment (zkevkev) [inline lib/stat_tracker.rb:1]: This method loos great. We just need to move all of this functionality over to the game class to ensure all the game functionality lives in the same place.

**Review by zkevkev — APPROVED** [review]
Thanks for your work on this Billy, this is a great start. Now we can just move this over to the game class and build from there.

File: lib/stat_tracker.rb
Comment (EricBelongea) [reply lib/stat_tracker.rb:1]: Agreed, I will create a new branch and add the percentages on to this new Game class.

**Pull request merged**